### DISCUSSION TITLE
Sprite generation tool

### DISCUSSION THREAD

**Discussion by ImperaEtConquer** [AlmasB/FXGL#1109] [Show and tell]
Hey, guys! I published a tool on itch.io a month ago. Decided to share it with you here too!

Thanks to the FXGL, it took me like 35~ minutes to develop it from scratch. I was shocked at how quickly and easily the whole process went.  

Yeah, it looks kinda ugly, but it does what it is supposed to do. 😅

It is also kind of funny that you can use FXGL to build tools that could be used to assist you in making a game on FXGL engine, lol. So keep that in mind, perhaps you might come up with some cool ideas too.

Here's the link: https://0a3r.itch.io/modern-interiors-character-generation-tool

And here are some stats if someone is interested. I even got several donations, didn't expect that at all
![image](https://user-images.githubusercontent.com/20888857/136862373-af9c1285-1694-44b4-b2a8-8188de766a8b.png)

**Comment by AlmasB** [comment]
Awesome stuff, very exciting to see FXGL being helpful for tool generation!
